src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -62,7 +62,7 @@
 from psyclone.parse.algorithm import Arg, parse
 from psyclone.parse.utils import ParseError
 from psyclone.psyGen import PSyFactory, InvokeSchedule, HaloExchange, BuiltIn
-from psyclone.psyir.nodes import colored, UnaryOperation, Reference
+from psyclone.psyir.nodes import colored, UnaryOperation, Reference, Routine
```

Comment:
dynamo0p3_test.py:4327:39: W0613: Unused argument 'tmpdir' (unused-argument)

Reply:
I added the compilation step and now it does use tmpdir.